docs/overview/formulation.rst

```diff
@@ -0,0 +1,163 @@
+.. _gs_formulation:
+
+============================================
+Formulation: Objectives and Quadratic Models
+============================================
+
+For quantum computing, as for classical, solving a problem requires that it
+be formulated in a way the computer and its software understand.
+
+For example, if you want your laptop to calculate the area of a $1 coin, you might
+express the problem as an equation, :math:`A=\pi r^2`, that you program as
+:code:`math.pi*13.245**2` in your Python CLI. For a laptop with Python software,
```

Comment:
"CLI" seems like an abbreviation at a totally different altitude than the rest of this section.

Reply:
--> terminal